frontend/src/components/common/podcast/podcast.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from './styles.module.scss';
+
+type Props = {
+  cover: string;
+  albumTitle: string;
+  singerName: string
+};
+
+const Podcast: React.FC<Props> = ({ cover, albumTitle, singerName }) => {
+
+  return (
+    <div className={styles.podcast}>
+      <div className={styles.imageBlock}>
+        <div className={styles.layer}> </div>
+        <img src={ cover } alt="cover"/>
```

Comment:
always add width/height/loading attrs

Reply:
I added the height and width attributes. Do I understand correctly that the last attribute is to add the "onLoad" attribute?

Reply:
https://web.dev/browser-level-image-lazy-loading/
loading="lazy"